test/torch/tensors/test_fv.py

```diff
@@ -121,31 +122,30 @@ def test_CoeffModulus_bfv_default(poly_modulus, SeqLevelType, result):
 
 
 @pytest.mark.parametrize(
-    "op1, op2, mod, result",
+    "op1, op2, coeff_mod, poly_mod, result",
     [
-        ([0, 0], [0, 0], 3, [0, 0]),
-        ([1, 2, 3, 4], [2, 3, 4, 5], 3, [0, 2, 1, 0]),
-        ([1, 2, 3, 4], [2, 3, 4, 5], 1, [0, 0, 0, 0]),
-        ([1, 2, 3, 4, 5], [1, -4], 3, [2, 1, 0, 1, 2]),
-        ([4, 4], [-4, -4, -4, -4], 4, [0, 0, 0, 0]),
+        ([0, 0], [0, 0], 3, 2, [0, 0]),
+        ([1, 2, 3, 4], [2, 3, 4, 5], 3, 4, [0, 2, 1, 0]),
+        ([1, 2, 3, 4], [2, 3, 4, 5], 1, 4, [0, 0, 0, 0]),
+        ([1, 2, 3, 4, 5], [1, -4], 3, 5, [2, 1, 0, 1, 2]),
+        ([4, 4], [-4, -4, -4, -4], 4, 4, [0, 0, 0, 0]),
     ],
 )
-def test_poly_add_mod(op1, op2, mod, result):
-    assert poly_add_mod(op1, op2, mod) == result
+def test_poly_add_mod(op1, op2, coeff_mod, poly_mod, result):
+    assert poly_add_mod(op1, op2, coeff_mod, poly_mod) == result
```

Comment:
Shouldn't poly_mod be an array of coefficients?

Reply:
https://github.com/OpenMined/PySyft/pull/3711#discussion_r446906200